Singer.API/ClientApp/src/app/modules/dashboard/components/shared-components/event-list/event-list.component.ts

```diff
@@ -27,11 +27,19 @@ export class HomeComponent implements OnInit {
          .subscribe(res => {
             this.availableLocations = res.items as SingerEventLocation[];
          });
-      this._authService.isAuthenticated$.subscribe(res => {
```

Comment:
Because there is already an authentication check happening in the dashboard component where this component is being used. Double authentication seemed uneccesary.

Reply:
clear, thanks